unit_tests/UnitTestTpetraHelperObjects.h

```diff
@@ -117,6 +130,11 @@ struct TpetraHelperObjectsBase {
             foundGoldCol = true;
             EXPECT_NEAR(vals[offset], constRowView.value(j), 1.e-14)<<"i: "<<i<<", j: "<<j;
           }
+          else if (!find_col(constRowView.colidx(j),
```

Comment:
@alanw0 Shouldn't this be `if (find_col(...`?

Reply:
Nope, it's checking the case where the column from the matrix is not found in the "gold" cols. In other words the column from the matrix is one of the "extra" graph columns which should correspond to a zero coefficient value.

Reply:
Ah, ok, got it.  `cols` are the "gold" columns.